lineman/app/components/invitation_form/team_link_modal/team_link_modal.coffee

```diff
@@ -0,0 +1,20 @@
+angular.module('loomioApp').factory 'TeamLinkModal', ->
+  templateUrl: 'generated/components/invitation_form/team_link_modal/team_link_modal.html'
+  controller: ($scope, Records, LoadingService, group, AppConfig, FlashService) ->
+    $scope.group = group
+    $scope.loading = true
+
+    $scope.load = ->
+      Records.invitations.fetchShareableInvitationByGroupId(group.id)
```

Comment:
that is sooo super weird.. must have fallen through the git cracks somehow. Thank you!